core/src/test/java/feign/client/TrustingSSLSocketFactory.java

```diff
@@ -43,7 +43,7 @@
   private static final Map<String, SSLSocketFactory> sslSocketFactories =
       new LinkedHashMap<String, SSLSocketFactory>();
   private static final char[] KEYSTORE_PASSWORD = "password".toCharArray();
-  private final static String[] ENABLED_CIPHER_SUITES = {"SSL_RSA_WITH_3DES_EDE_CBC_SHA"};
+  private final static String[] ENABLED_CIPHER_SUITES = {"TLS_RSA_WITH_AES_256_CBC_SHA"};
```

Comment:
This is a java 8 only change and your PR is against our 9.x branch, which is not java 8 compatible.  Please break this out into another PR against the `master` branch.